Sources/LeafKit/LeafToken.swift

```diff
@@ -73,7 +105,7 @@ public indirect enum ParameterDeclaration: CustomStringConvertible {
         case .parameter(let p):
             return p.short
         case .expression(let p):
-            return p.map { $0.short }.joined(separator: " ")
+            return "[\(p.map { $0.short }.joined(separator: " "))]"
```

Comment:
Minor change to internal .short description on expressions to wrap in brackets for clarity - eg
`expression([2 + 4] * [4 - 2])` instead of `expression(2 + 4 * 4 - 2)`